addons/event/migrations/12.0.1.0/post-migration.py

```diff
@@ -1,18 +1,21 @@
 # Copyright 2018 Eficent <http://www.eficent.com>
 # Copyright 2019 Tecnativa - Pedro M. Baeza
+# Copyright 2019 Tecnativa - Jairo Llopis
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
-from openupgradelib import openupgrade
+from openupgradelib import openupgrade, openupgrade_120
 
 
-@openupgrade.migrate(use_env=False)
-def migrate(cr, version):
+@openupgrade.migrate()
+def migrate(env, version):
     openupgrade.load_data(
-        cr, 'event', 'migrations/12.0.1.0/noupdate_changes.xml')
+        env.cr, 'event', 'migrations/12.0.1.0/noupdate_changes.xml')
```

Comment:
@Yajo for reducing diff, put a first line with `cr = env.cr` and don't touch the rest

Reply:
Is it worth it to add a variable just for the sake of reducing diff? :roll_eyes:

Reply:
Yes, and check Travis that is red